src/tls13.c

```diff
@@ -5359,6 +5356,7 @@ int DoTls13ServerHello(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
         DtlsCIDOnExtensionsParsed(ssl);
 #endif /* WOLFSSL_DTLS_CID */
 
+    inOutIdxCopy = *inOutIdx;
```

Comment:
Let's use the inOutIdx argument passed in later with DoServerHello like the previous calls in this function. And just not update the inOutIdx in the next line if it's not TLS version 1.3 (which should be known now after checking the extensions). This gets rid of the need for a new local argument.